bin/docker/Dockerfile_ubuntu

```diff
@@ -5,7 +5,7 @@ ENV FC_LANG=en-US LC_CTYPE=en_US.UTF-8
 # Dependencies
 RUN apt-get update && \
   apt-get upgrade -y && \
-  apt-get install -y ca-certificates ca-certificates-java fonts-dejavu && \
+  apt-get install -y ca-certificates ca-certificates-java fonts-noto && \
```

Comment:
This is the only difference between this PR and [the original PR](https://github.com/metabase/metabase/pull/31764/files#diff-93de86efbdc50b8923fb11ace6d2d19ce641c29e672ce0be97be8663b9aaded6R8)

Previously it was
```sh
apt-get install -y ca-certificates ca-certificates-java fonts-noto fonts-noto-arabic fonts-noto-hebrew fonts-noto-cjk && \
```

I tried to find `font-noto-hebrew` in apt, but it seems it was never there. It was in apk instead, and looking at the [files from that package](https://pkgs.alpinelinux.org/contents?branch=edge&name=font%2dnoto%2dhebrew&arch=x86&repo=community), they are already included in [fonts-noto-extra](https://packages.ubuntu.com/jammy/all/fonts-noto-extra/filelist), which is installed as a recommended package for `fonts-noto` anyway. The same goes for [fonts-noto-arabic](https://pkgs.alpinelinux.org/contents?branch=edge&name=font%2dnoto%2darabic&arch=s390x&repo=community).